project/profiles/capacity4more/modules/c4m/admin/c4m_admin_projects/c4m_admin_projects.module

```diff
@@ -0,0 +1,26 @@
+<?php
+/**
+ * @file
+ * Code for the Admin Projects feature.
+ */
+
+include_once 'c4m_admin_projects.features.inc';
+
+/**
+ * Implements hook_menu().
+ */
+function c4m_admin_projects_menu() {
+  $items = array();
+
+  $items['admin/c4m/projects'] = array(
+    'title' => 'Projects',
+    'description' => 'Administration interface to manage projects & their status.',
+    'page callback' => 'views_embed_view',
```

Comment:
I guess the purpose of this is to create a menu item and a subitem. admin/c4m/projects vs admin/c4m/projects/pending